web_src/js/features/repo-issue-list.js

```diff
@@ -8,6 +8,7 @@ import {createSortable} from '../modules/sortable.js';
 import {DELETE, POST} from '../modules/fetch.js';
 
 function initRepoIssueListCheckboxes() {
+  if (!window.config.isSignedIn) return;
   const issueSelectAll = document.querySelector('.issue-checkbox-all');
   const issueCheckboxes = document.querySelectorAll('.issue-checkbox');
```

Comment:
This looks even more hacky than
```suggestion
  const issueSelectAll = document.querySelector('.issue-checkbox-all');
  const issueCheckboxes = document.querySelectorAll('.issue-checkbox');
  if(!issueSelectAll) return;
```

Reply:
TBH, I also prefer this one, instead of adding the "isSignedIn" now.

(Maybe the "isSignedIn" could be useful some cases, it could be added when it is really needed ....)